README.md

```diff
@@ -66,18 +66,54 @@ To achieve this MLOS provides:
 
     Once hooks are created in the target system, iteration on the external agent can be more rapidly developed and deployed.
 
-## Build
+
+## Python only installation
+
+Some of the examples require only the installation of the mlos Python library. These examples do not use the shared memory infrastructure.
+To use this simplified installation, it's recommended to use the Anaconda python distribution and create a new conda environment:
+
+```
+$ conda create -n mlos_environment
+$ conda activate mlos_environment
+```
+
+You can download the mlos code using git:
+
+```
+$ git clone git@github.com:microsoft/MLOS.git
+```
+
+To install the Python library, change to the Python directory and install with ``pip``:
+
+```
+$ cd MLOS/source/Mlos.Python
+$ pip install -e .
+```
+
+You can also install the package directly without checking out the code:
+
+```
+$ pip install "git+https://github.com/microsoft/MLOS.git#egg=mlos&subdirectory=source/Mlos.Python"
+```
+
+However, this does not include the examples.
+
+After this installation, you can run any of the Python-only example notebooks.
+A good place to start is the [introduction to Bayesian Optimization](#).
```

Comment:
This seems incomplete.

"A good place to start the [I{captial I?}ntroduction to Bayesian Optimization](# a link should go here?) notebook (missing word?)."

Reply:
Adding "notebook" might be more explicit but saying "A good place to start is the Introduction to Bayesian Optimization Notebook" is a complete sentence, right? Is it not clear?

Reply:
My comment was more about the fact that the link looked incomplete.

The rest of the wording tweaks you could take or leave, they were just minor nits.

Reply:
ah ok. Yes, that required the PR to be merged that you just merged :)

Reply:
done :)